pkgs/by-name/vl/vlc-bittorrent/package.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, autoconf-archive
+, autoreconfHook
+, boost
+, fetchFromGitHub
+, libtool
+, libtorrent-rasterbar
+, libvlc
+, openssl
+, pkg-config
+, stdenv
+}:
+
+# VLC does not know where the vlc-bittorrent package is installed.
+# make sure to have something like:
+#   environment.variables.VLC_PLUGIN_PATH = "${pkgs.vlc-bittorent}";
+
+stdenv.mkDerivation rec {
+  pname = "vlc-bittorrent";
+  version = "2.15.0";
+
+  src = fetchFromGitHub {
+    owner = "johang";
+    repo = pname;
+    rev = "6810d479e6c1f64046d3b30efe78774b49d1c95b";
+    hash = "sha256-7FHeQYHbMKZJ3yeHqxTTAUwghTje+gEX8gSEJzfG5sQ=";
+  };
+
+  nativeBuildInputs = [
+    autoconf-archive
+    autoreconfHook
+    pkg-config
+  ];
+
+  buildInputs = [
+    boost
+    libtool
+    libtorrent-rasterbar
+    libvlc
+    openssl
+  ];
+
+  meta = with lib; {
+    description = "A bittorrent plugin for VLC.";
```

See the [`meta` attributes documentation](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes):

```suggestion
    description = "A bittorrent plugin for VLC";
```